lib/msf/core/modules/external/bridge.rb

```diff
@@ -72,6 +72,8 @@ def send(message)
         else
           raise "Error running module #{self.path}"
         end
+      rescue ::Exception => e
+        raise handle_exception(e)
```

Comment:
I would a default implementation so we get the original error instead of a `NoMethodError` when a non-customized module fails.